app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncResponseModel.java

```diff
@@ -0,0 +1,14 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.time.LocalDateTime;
+
+import lombok.Data;
+
+@Data
+public class SenderSyncResponseModel {
```

Comment:
Name should be SyncResponseModel because it is not specific to the sender

Reply:
Alright. Renamed.